modules/xpfunmac.cmake

```diff
@@ -1024,6 +1020,10 @@ macro(xpSourceListAppend)
   else()
     list(REMOVE_DUPLICATES masterSrcList)
     if(EXISTS ${CMAKE_SOURCE_DIR}/.git)
+      if(NOT GIT_FOUND)
+        include(FindGit)
+        find_package(Git)
+      endif()
```

Comment:
This ended up saving about 2-4 seconds since the code that followed was finding git over and over again.